doc/cmake.txt

```diff
@@ -209,6 +210,32 @@ cq			Close the CMake console window.
 
 <C-C>			Stop the current command.
 
+==============================================================================
+Events							*cmake-events*
+
+To customize behaviour after a command has finished, Vim-CMake calls user
+autocommands
+
+List of events~
+
+CMakeBuildFailed
+CMakeBuildSucceeded
+
+Example usage of CMakeBuildFailed to jump to the first error
+>
+      let g:jump_on_error = 0 " We do not want to focus the console
```

Comment:
Indent all lines with a single tab character please

Reply:
Or I actually mean start all lines with one tab character, and add e.g. two or four spaces (or another tab character) for indented lines